.github/workflows/cache-issues.yaml

```diff
@@ -0,0 +1,29 @@
+name: Report remaining open issues
+on:
+  schedule:
+    # Daily at 8:20 UTC
+    - cron: '20 8 * * *'
```

Comment:
gh issue list doesn't use many requests (1 per 100 issues) and are resetting every hour. Maybe run it hourly?

Reply:
I didn't yet really setup it, I've trying to set it up correctly
and credentials is painful

Reply:
it's copy paste from github examples...

and I think we still gonna do it daily